app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -384,8 +384,15 @@ class Meta:
         ]
 
     def __init__(self, instance=None, data=None, **kwargs):
-        self.should_call_preview_task = (
-            data.get("status") == NimbusExperiment.Status.PREVIEW
+        self.should_call_preview_task = instance and (
+            (
+                instance.status == NimbusExperiment.Status.DRAFT
+                and data.get("status") == NimbusExperiment.Status.PREVIEW
+            )
+            or (
+                instance.status == NimbusExperiment.Status.PREVIEW
+                and data.get("status") == NimbusExperiment.Status.DRAFT
+            )
         )
```

Comment:
I don't think we need this level of rigor here for two reasons

1) The task itself is idempotent so calling it an extra time doesn't hurt anything
2) I think the only time we want to fast forward this call is when it's moving into preview since the user will want to test without waiting (maybe), but moving back to draft and unpublishing it doesn't block anyones workflow so that can wait until the next scheduled call

Regardless, if we want to add these additional checks here I would expect the tests to be expanded to test each of these cases.